forums/views.py

```diff
@@ -95,155 +116,170 @@ def account_profile(request, username):
             user.save()
             form_data = form.save(commit=False)
             form_data.user_id = user.id
-            
-            # if 'picture-clear' in request.POST and request.POST['picture-clear']:
+
+            # if 'picture-clear' in request.POST and
+            # request.POST['picture-clear']:
             #     #if not old_file == new_file:
             #     if os.path.isfile(old_file_path):
             #         os.remove(old_file_path)
             #
             # if 'picture' in request.FILES:
             #     form_data.picture = request.FILES['picture']
-            
-            #form_data.save()
-            #profile.address = address
-            #profile.phone = phone
+
+            # form_data.save()
+            # profile.address = address
+            # profile.phone = phone
             profile.save()
-            
+
             """if 'picture' in request.FILES:
```

Comment:
This is a huge blob of commented code. It already seems that this is unnecessary